### DISCUSSION TITLE
Rack aware change

### DISCUSSION THREAD

**Discussion by sreejesh-radhakrishnan-db** [strimzi/strimzi-kafka-operator#8095] [Q&A]
Hi 

My current KAFKA cluster is setup with NO RACK AWARE setup l interestingly ZK has its STORAGE created across three ZONE on my datacentre. But    KAFKA has all its storage created on same ZONE. (i.e ZONE-c data center) 

By Q. if I change the KAFA manifest and redeploy with below rack aweare setting. will Operator be able to move the current Storage PVC from single zone to 3-zones?

   rack:
      topologyKey: topology.kubernetes.io/zone

**Comment by scholzj** [comment]
No, it will not move the storage. That is not really possible. The only way to move the storage is basically to _create a new storage_ and move the data.
